node/modules/storageminer.go

```diff
@@ -568,3 +569,90 @@ func NewGraphqlServer(lc fx.Lifecycle, prov *storagemarket.Provider, dealsDB *db
 
 	return server
 }
+
+func BasicDealFilter(cfg config.DealmakingConfig, userCmd dtypes.StorageDealFilter) func(onlineOk dtypes.ConsiderOnlineStorageDealsConfigFunc,
```

Comment:
nit: suggest putting this in a separate file so that storageminer.go doesn't get too bloated